src/Mysqli/MysqliStatement.php

```diff
@@ -301,27 +301,22 @@ public function bindParam($parameter, &$variable, string $dataType = ParameterTy
      */
     private function bindValues(array $values)
     {
-        $params = [];
         $types  = str_repeat('s', \count($values));
 
-        if (!empty($this->parameterKeyMapping)) {
+        if ($this->parameterKeyMapping !== []) {
             foreach ($values as $key => &$value) {
```

Comment:
```suggestion
            foreach ($values as $key => $value) {
```
Should this be changed, too? It has landed here with my conflict resolution due to the merged PR #351 .

Reply:
Seems I made another mistake when resolving the conflict as the `$params` variable is not defined with the changes from this PR here?

Reply:
I have rebased. It should be ok now.

Reply:
Thanks.